ArchiSteamFarm/Steam/Bot.cs

```diff
@@ -3154,7 +3154,7 @@ public sealed class Bot : IAsyncDisposable {
 				(string? key, string? name) = BotDatabase.GetGameToRedeemInBackground();
 
 				if (string.IsNullOrEmpty(key) || string.IsNullOrEmpty(name)) {
-					ArchiLogger.LogNullError($"{nameof(key)} || {nameof(name)}");
+					ArchiLogger.LogNullError(null, $"{nameof(key)} || {nameof(name)}");
```

Comment:
We must change it then, I suggest splitting into two `if` checks and two standalone log nulls.